bigtable/client/client_options_test.cc

```diff
@@ -46,3 +46,71 @@ TEST(ClientOptionsTest, EditCredentials) {
   EXPECT_EQ(typeid(grpc::InsecureChannelCredentials()),
             typeid(client_options_object.credentials()));
 }
+
+TEST(ClientOptionsTest, SetGrpclbFallbackTimeout) {
+  bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
+  client_options_object.SetGrpclbFallbackTimeout(5);
+  grpc::ChannelArguments c_args = client_options_object.channel_arguments();
+  grpc_channel_args test_args = c_args.c_channel_args();
+  EXPECT_EQ((size_t)2, test_args.num_args);
+  EXPECT_EQ(5, test_args.args[1].value.integer);
+}
+
+TEST(ClientOptionsTest, SetCompressionAlgorithm) {
+  bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
+  client_options_object.SetCompressionAlgorithm(GRPC_COMPRESS_NONE);
+  grpc::ChannelArguments c_args = client_options_object.channel_arguments();
+  grpc_channel_args test_args = c_args.c_channel_args();
+  EXPECT_EQ((size_t)2, test_args.num_args);
```

Comment:
s/(size_t)2/2UL/

In general we prefer `static_cast<size_t>(x)`, to C-style casts (`(size_t)x`),  or to constructor-like casts (`size_t(x)`).

Reply:
Done.